src/sentry/seer/anomaly_detection/get_anomaly_data.py

```diff
@@ -34,7 +34,7 @@ def get_anomaly_data_from_seer(
     aggregation_value: float | None,
 ) -> list[TimeSeriesPoint] | None:
     snuba_query = alert_rule.snuba_query
-    if not snuba_query or aggregation_value is None:
+    if not snuba_query or aggregation_value in (None, "NULL_VALUE"):
```

Comment:
```suggestion
    if not snuba_query or aggregation_value in {None, "NULL_VALUE"}:
```
the other was a tuple, rather than a set. Also, thoughts on making this a constant?